charts/cadc-tap/templates/serviceaccount.yaml

```
@@ -0,0 +1,15 @@
+{{- if or .Values.serviceAccount.create .Values.cloudsql.enabled -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Values.serviceAccount.name }}
```

I don't think there's a great default value for the service account name, so I would be inclined to do the same `required` thing here. So:
```suggestion
  name: {{ required "serviceAccount.name must be set" .Values.serviceAccount.name | quote }}
```

---

Okay i've made this required